src/Avalonia.Themes.Default/ComboBox.xaml

```diff
@@ -24,13 +25,21 @@
     <Setter Property="VerticalContentAlignment" Value="Center" />
     <Setter Property="Padding" Value="4" />
     <Setter Property="MinHeight" Value="20" />
+    <Setter Property="PlaceholderForeground" Value="{DynamicResource ComboBoxPlaceHolderForeground}" />
```

Comment:
I am not sure, if ComboBoxPlaceHolderForeground exists in the default theme

Reply:
https://github.com/AvaloniaUI/Avalonia/search?q=ComboBoxPlaceHolderForeground&unscoped_q=ComboBoxPlaceHolderForeground

Reply:
It works on ControlCatalog, because it has all resources from both themes, when default it enabled.

Reply:
Yep, probably my bad, let me check in a separate app

Reply:
Hi @maxkatz6,
Yes you are absolutely right about resource name, that line should look like ` <Setter Property="PlaceholderForeground" Value="{DynamicResource ThemeForegroundBrush}" />`. Also I noticed that PlaceholderForegroundProperty is not used anywhere, so I will add TemplateBinding for it and open another PR